src/components/Menu.module.scss

```diff
@@ -1,42 +1,39 @@
 .Menu {
   cursor: pointer;
-  
-  @media (hover: hover) and (pointer: fine) {
-    &:hover {
-      background-color: grey;
-    }
-  }
-  
-  .icon {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    height: 3em;
-    font-weight: bold;
-    aspect-ratio: 1 / 1;
-  }
 
   .items {
-    display: none;
-    flex-direction: column;
-    row-gap: 10px;
+    display: flex;
+    flex-direction: row;
+    column-gap: 25px;
     color: red;
-    position: absolute;
     background-color: #2F2F2F;
-    padding: 10px;
+    padding: 25px;
     z-index: 1;
   }
-  
-  &.open {
-    .items {
-      display: flex;
-    }
-  }
 }
 
 .MenuItem {
+  text-decoration: none;
   color: white;
-  text-decoration: unset;
-  cursor: pointer;
-  font-size: 20px;
+  position: relative;
+  z-index: 1;
+  font-size: 25px;
+
+  &::after {
+    content: "";
+    background: rgba(200, 200, 200, 1);
+    width: calc(100% + 6px);
+    height: 2px;
+    position: absolute;
+    bottom: -2px;
+    left: -3px;
+    transition: all .15s cubic-bezier(0.445, 0.05, 0.55, 1);
+    border-radius: 5px;
+    z-index: -1;
+  }
+
+  &:hover::after {
+    height: calc(100% + 4px);
+    background: rgba(200, 200, 200, 0.5);
```

Comment:
Instead of animating `height` and `background`, you should animate `transform` and `opacity` so that the animation can be [run on the compositor](https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count).